.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Setup NodeJs
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build & Verify
+        run: yarn verify
+
+      - name: Setup git
+        run: |
+          git config user.name "Release Bot"
+          git config user.email noreply@github.com
+
+      - name: Release
+        run: semantic-release --dry-run
```

Comment:
Remove `--dry-run` once we're sure everything is ready - do this in a future PR.